code/global.dm

```diff
@@ -28,9 +28,9 @@ var/world_qdel_log
 var/href_logfile
 var/station_name        = "CEV Eris"
 var/station_short       = "Eris"
-var/const/dock_name     = "N.A.S. Crescent"
-var/const/boss_name     = "Central Command"
-var/const/boss_short    = "Centcom"
+var/const/dock_name     = "NTV Faith"
+var/const/boss_name     = "Hanza Trade Union"
```

Comment:
```suggestion
var/const/boss_name     = "Hansa Trade Union"
```

Reply:
Hansa is like, entire Hansa. Might want to specify it a little, like Hansa Headquarters or something